README.md

```diff
@@ -73,112 +72,113 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Added support for port numbers in the URI plugin. Thanks [@KohGeek](https://github.com/KohGeek)!
-- Fixed query errors when the search delay option was turned off.
-- New History plugin to search for old search results. Thanks [@jefflord](https://github.com/jefflord)!
-- Changed the default TimeDate activation keyword to `)`, as queries starting by `(` are expected as Calculator global queries, and added information in Settings so users know that some activation keywords may conflict with normal usage of some plugins when trying to do a global query. Thanks [@htcfreek](https://github.com/htcfreek)!
-- The Unit Converter plugin updated its UnitsNet dependency and now supports plural units. Thanks [@FWest98](https://github.com/FWest98)!
-- Improved the validation logic in the Calculator plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Improved the keyboard hooks performance. (This was a hotfix for 0.62)
+- Fixed a bug that was causing Quick Accent to interfere with Keyboard Manager. (This was a hotfix for 0.62)
+- Added the correct ß uppercase character. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Accent character selection should now wrap around. Thanks [@wmentha](https://github.com/wmentha)!
+- Added language selection setting to reduce the number of accented characters shown. Thanks [@damienleroy](https://github.com/damienleroy)!
 
-### Runner
+### Screen Ruler
 
-- Improved: Clean up old install folders and logs at startup. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Improved UI/UX and settings descriptions.
+- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
 
 ### Settings
 
-- Image and phrasing adjustments.
-- Icon and image updates for the new utilities. Thanks [@niels9001](https://github.com/niels9001)!
+- UI icons updated. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+- Descriptions improvement and disambiguation. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+- Fixed checkbox margins and other design tweaks. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
 
-### Shortcut Guide
+### Text Extractor
 
-- Fixed the Narrator shortcut to include the newly added Control key.
+- Removed extra spaces when recognizing Chinese, Japanase or Korean languages. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
+- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
+- Fixed an issue where a selection would start on right-click.
 
 ### Installer
 
-- Fixed a regression that was causing the PowerToys shortcut to be deleted on update. (This was a hotfix for 0.61)
-- Updated the .NET dependency to 6.0.8.
+- Added logic to exit PowerToys on upgrade before trying to update .NET.
+- Updated the .NET dependency to 6.0.9.
+- Added clearer installation step names for the boostrapper.
```

Comment:
@yuyoyuppe & @Jay-o-Way :)

Reply:
@jay-o-way
not only you helped here.

Reply:
@yuyoyuppe is part of the core team :) 
I'll add @Jay-o-Way  and @htcfreek , though.

Reply:
I've pushed the suggested changes.